lib/pensio_api/mixins/request_defaults.rb

```diff
@@ -9,7 +10,7 @@ def self.included(base)
 
       HEADERS = {
         'Content-Type' => 'application/x-www-form-urlencoded; charset=utf-8',
-        'x-altapay-client-version' => "RUBYSDK/#{PensioAPI::VERSION}"
+        'x-altapay-client-version' => 'RUBYSDK/' + PensioAPI::AltapayVersion.getVersion
```

Comment:
Why on earth would you do this? This is demonstrably worse. String interpolation (as we had it) is preferable to concatenation.